### DISCUSSION TITLE
New Language Services support in Bot Framework Composer (custom question answering + conversational language understanding)

### DISCUSSION THREAD

**Discussion by soydiloreto** [microsoft/BotFramework-Composer#9361] [Design]
In a few weeks (Octuber 1st, 2022), QnA Maker will no longer support new QnA Maker resources. We will have to use the new services of "[Azure Cognitive Services for Language](https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/overview)". This new services are not yet supported by Bot Framework Composer, and there is no news or estimated dates for this support to be guaranteed. Could I get information about the future of Bot Framework Composer and the support for these new Azure services? There are many clients who have chosen to use this no-code IDE and a few weeks before the end of the QnA Maker cycle (new resources), there is no clear information.

In other discussions ([here](https://github.com/microsoft/BotFramework-Composer/issues/9190) and [here](https://github.com/microsoft/BotFramework-Composer/issues/8896)), there is no clear answer on this topic.

#8896 #9190

**Comment by jaydeepn** [comment]
Hi @pablodiloreto - please take a look at this post: https://github.com/microsoft/BotFramework-Composer/issues/9400
